### PR TITLE
Some edits

### PR DESCRIPTION
**Pull request by erikjalevik**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -441,7 +444,7 @@ Whenever you want to run software on an actual device (as opposed to the simulat
 
 Besides certificates, there are also __provisioning profiles__, which are basically the missing link between devices and certificates. Again, there are two types to distinguish between development and distribution purposes:
 
-* __Development provisioning profile:__ It contains a list of all devices that the device is authorized to be built on. It is also linked to one or more development certificates, one for each developer that is allowed to use the profile. The profile can be tied to a specific app, but for most development purposes it's perfectly fine to use the wildcard profile, whose App ID ends in an asterisk (*).
+* __Development provisioning profile:__ It contains a list of all devices that the project is authorized to be built on. It is also linked to one or more development certificates, one for each developer that is allowed to use the profile. The profile can be tied to a specific app, but for most development purposes it's perfectly fine to use the wildcard profile, whose App ID ends in an asterisk (*).
```
Comment (richeterre) [inline README.md]: Good catch – perhaps "that are authorized to install and run the software" would be even better?

Comment (erikjalevik) [inline README.md]: Yeah absolutely, feel free to change, wasn't quite sure about the fix to be honest, but I at least knew that a device can not be built on a device.

Comment (richeterre) [inline README.md]: Can you make that commit? I'd have to go through the whole branch checkout process to add stuff to this PR…

Comment (erikjalevik) [inline README.md]: Done.

**Comment by richeterre** [comment]
Looks good to me! :rocket:

**Pull request merged**